src/types/games/wows/maps.ts

```diff
@@ -0,0 +1,361 @@
+import { Map } from '@/store/types'
+
+export default class WowsMaps {
```

Comment:
Hpefully these are fetched via API, not via this file.

Reply:
@Eirmas did this work, I think it was done because
1) The maps rarely change and we will already be updating maps as they change because it is a manual process - unfortunately
2) He needs to map in extra properties which would result in a similar amount or more code.

Please talk to him Niko but don't hold this PR up because of it. We need a new ticket to cover the work